x-pack/libbeat/docs/aws-credentials-config.asciidoc

```diff
@@ -0,0 +1,77 @@
+= AWS Credentials Configuration
```

Comment:
A level one heading (=) is not allowed here because you are including it under a level 2. I'd suggest changing this to level 3 (===) and the nested sections to level 4 (====). Then you'll get it building and can view the output. There are other structural issues that you'll need to address. Reach out to me if you need more help.

Reply:
You can use [float] tags before the heading to keep the content in the section on the same page as the previous section.

Reply:
Will do! Thanks!